src/devices/Mcp23xxx/Mcp23x1x.cs

```diff
@@ -54,5 +54,25 @@ protected Mcp23x1x(BusAdapter device, int reset, int interruptA, int interruptB,
         /// Reads the interrupt pin for the given port if configured.
         /// </summary>
         public PinValue ReadInterrupt(Port port) => InternalReadInterrupt(port);
+
+        /// <summary>
+        /// Reads all bits of port A in a single operation.
+        /// </summary>
+        /// <returns></returns>
```

Comment:
what does it returns? I guess the 8 bits on the first byte but -1 if failure? Correct?

Reply:
Good catch. Shouldn't StyleCop complain about an empty returns tag???

Reply:
Not sure we've configured it for this!